### ISSUE TITLE
Rassberry pi 4  . ./btcpay-setup.sh -i error

### ISSUE THREAD

**Issue by mbio16** [btcpayserver/btcpayserver#2404]
Hi, 

When running `. ./btcpay-setup.sh -i` in /btcpayserver-docker as root. Errors connected with python occur:

```BTCPay Server starting... this can take 5 to 10 minutes...
BTCPay Server started
Traceback (most recent call last):
  File "docker-compose", line 2, in <module>
  File "/usr/local/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "compose/cli/main.py", line 13, in <module>
ModuleNotFoundError: No module named 'docker'
[12298] Failed to execute script docker-compose
Traceback (most recent call last):
  File "docker-compose", line 2, in <module>
  File "/usr/local/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
  File "compose/cli/main.py", line 13, in <module>
ModuleNotFoundError: No module named 'docker'
[12300] Failed to execute script docker-compose
```
docker servis is running

**Comment by pavlenex** [comment]
Which OS are you using for RPI4, where are you following instructions for installing btcpay on RPI?
It looks like you may need to upgrade python.
Please don't ignore the issue template, as it gives us an insight into your environment and can help us understand your issue better.

**Comment by mbio16** [comment]
Hi, I followed all instruction for RPI 4 ona that are in docs. First, I tried with Raspberry Pi OS and then with Ubuntu Server 20.04.2 LTS for RPI. Same result. My python3 version is Python 3.8.5.

**Comment by pavlenex** [comment]
Perhaps try on 20.04, not LTS. I haven't tried it on 20.04.2, TBH. Still, it seems like your docker isn't running for some reason.

**Comment by pavlenex** [comment]
Perhaps try downgrading ubuntu, I am quite certain 20.04 should work.
I haven't tried it on 20.04.2, TBH. Still, it seems like your docker isn't running for some reason.
I think this question is more likely to get a proper answer in #RPI channel on chat.btcpayserver.org

**Comment by mbio16** [comment]
Ok, will provide downgrade. However, it is wierd that the same happend on Raspberry Pi OS.

**Comment by pavlenex** [comment]
When was the last time you tried this, we had a fix in btcpayserver-docker 20 hours ago, but I doubt that's it. RPI people should know to answer this better if what I suggested doesn't work.

**Comment by mbio16** [comment]
about an 1 hour ago on Ubuntu and on Thursday on RPI OS.

**Comment by pavlenex** [comment]
Hi, @mbio16 there was a bug in docker deployment which is fixed in #2429 sorry about that. Can you give it a try again and let us know?

**Comment by mbio16** [comment]
> thanks for letting me know. Will try.
